### PR TITLE
Updated Client Facing Documentation for SDK 

### PR DESCRIPTION
**Pull request by ajay-ragu**
Updated Client Facing Documentation for SDK for Android
- Added heading for standard unit and updated table

### CODE REVIEW

**Review by m-viegas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,8 +28,20 @@ The SDK does this by providing a custom view called `FlyerView`, which has 2 mai
 1. Renders the NativeX experience 
 2. Provides callback methods for your app to handle events from the `FlyerView`
 
+## Standard Unit <a name="standard-unit"></a>
+The standard 300x1800 ratio ad unit designed to be placed at the bottom of an article
+
 <img src="./assets/SDKIntegration.png" width="25%">
 
+## Compact Unit <a name="compact-unit"></a>
+The compact unit appears in a smaller size of 300x600 and is intended for to be placed in the middle of the page. This ad type can automatically expand as the user engages with the content. To use the compact mode, it is recommended to toggle on both `startCompact` and `dwellExpand`.[Read more about these options](#options)
```
Comment (Mtriebeg) [inline README.md:37]: "...it is recommended to toggle on both" isn't accurate, we should be descriptive. Suggest " To use the compact mode, define `startCompact` and `dwellExpand` as `true`. [Read more about these options](#options)"

File: README.md [hunk 2]
```diff
@@ -208,3 +224,10 @@ flyerView.setFlyerEventsListener(object: FlyerEventsListener() {
 ```
 
 Please note that the callbacks are optional and you do not have to listen to them. You can also override only the needed methods.
+
+### Options <a name="options"></a>
```
Comment (Mtriebeg) [inline README.md:228]: Update this section to match iOS's

File: README.md [hunk 3]
```diff
@@ -95,6 +110,7 @@ or from your `MainActivity`.
 - ``isReactNative`` - A boolean property indicating if the SDK is used in an React Native app
 - ``zoneIds`` - (optional) An array of zones used to separate site/properties within a media partner's portfolio to facilitate content targeting. For example, a zone for sports, another for news or food.  ZoneIDs are constrained to integer values and must align with the ID's provided by Flipp
 - ``contentCode`` - (optional) A string that can be used to invoke different campaign content for testing purposes
+- ``options`` - (optional) A set of SdkOptions that can be used to configure the experience
```
Comment (Mtriebeg) [inline README.md:113]: Update to match iOS section

File: README.md (hunk 1 above)
Comment (Mtriebeg) [reply README.md:37]: Check casing of startCompact` and `dwellExpand` for Android, not camel case?

File: README.md (hunk 3 above)
Comment (ajay-ragu) [reply README.md]: @m-viegas  to confirm, update to the following?

- ``options`` - (optional) A Set of loader options that can be used to customize how the ad renders. [See available options](#options)

Comment (m-viegas) [reply README.md]: @ajay-ragu - I've updated it

**Pull request merged**